### ISSUE TITLE
Make it easy to install Mathesar when not using install.sh

### ISSUE THREAD

**Issue by silentninja** [mathesar-foundation/mathesar#2790]
These steps make the build from source installation a little easier.
- [ ] Set up a GitHub action that automatically builds static files when we create a release
  - [ ] Update documentation to remove building static files manually.

**Comment by silentninja** [comment]
Blocked by #2427

**Comment by kgodey** [comment]
Closing, requirements out of date. Also seems like a duplicate of https://github.com/mathesar-foundation/mathesar/issues/3163
